.mpsrc

```diff
@@ -30,11 +30,11 @@
     "distributed_kv_name": "HashiCorpConsul",
```

Comment:
are any of these being used anymore:     
"distributed_kv_ip": "localhost",
    "distributed_kv_port": "8500",
    "distributed_kv_name": "HashiCorpConsul",
    "web_proxy_port": "8100",
    "network_adaptor": "eth0",

Reply:
done